markdown/extensions/admonition.py

```diff
@@ -95,3 +96,4 @@ def get_class_and_title(self, match):
 
 def makeExtension(*args, **kwargs):
     return AdmonitionExtension(*args, **kwargs)
+
```

Comment:
You broke linting with this extra line. There should only be one newline at the end of files, not two.